opensrp-opd/src/main/java/org/smartregister/opd/utils/Utils.java

```diff
@@ -0,0 +1,61 @@
+package org.smartregister.opd.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.smartregister.Context;
+import org.smartregister.opd.OpdLibrary;
+import org.smartregister.opd.R;
+import org.smartregister.opd.pojos.OpdMetadata;
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by ndegwamartin on 25/02/2019.
+ */
+public class Utils extends org.smartregister.util.Utils {
+    public static final ArrayList<String> ALLOWED_LEVELS;
+    public static final String DEFAULT_LOCATION_LEVEL = "Health Facility";
+    public static final String FACILITY = "Facility";
+
+
+    static {
+        ALLOWED_LEVELS = new ArrayList<>();
+        ALLOWED_LEVELS.add(DEFAULT_LOCATION_LEVEL);
+        ALLOWED_LEVELS.add(FACILITY);
+    }
+
+    public static int getProfileImageResourceIDentifier() {
+        return R.mipmap.ic_launcher;
+    }
+
+
+
+
+    public static Date dobStringToDate(String dobString) {
+        DateTime dateTime = dobStringToDateTime(dobString);
+        if (dateTime != null) {
+            return dateTime.toDate();
+        }
+        return null;
+    }
+
+    public static DateTime dobStringToDateTime(String dobString) {
+        try {
+            if (StringUtils.isBlank(dobString)) {
+                return null;
+            }
+            return new DateTime(dobString);
+
+        } catch (Exception e) {
+            return null;
```

Comment:
Does this need to be logged and/or traceable?

Reply:
yeah